### ISSUE TITLE
Is attention-mask needed in pre-training in tokenize_function_seq_desc?

### ISSUE THREAD

**Issue by zwanah** [QizhiPei/BioT5#4]
While padding="max_length", truncation=True, I feel that it would be better to use an attention mask here, right?

**Comment by QizhiPei** [comment]
Hello:

This function is only used for biotranslation task in pre-training and will not be used in fine-tuning.

**Comment by zwanah** [comment]
Sorry，I mean pre-training

**Comment by QizhiPei** [comment]
Thank you for bringing this to our attention! You're right about the need for an attention mask for biotrainslation task in pre-training. This oversight is a bug in our code. 

We appreciate your contribution in identifying this issue, and we will work on fixing it promptly. Your feedback helps us improve. Thanks again!

**Comment by QizhiPei** [comment]
I add attention_mask to batch in the newest commit.
Thanks for your kindly help~
